configuration-resources/large-language-models-llms/vertex-configuration.md

```diff
@@ -1,21 +1,29 @@
 # vertex-configuration
 
-A Google vertex configuration.
+A vertex-configuration resource is used for AI agents.
+
+{% hint style="warning" %}
+Note that the following examples sets sensitive data as plain text. This is for education purpose only, it's highly recommended to use [secrets](../../building-applications/secrets.md).
+{% endhint %}
+
 
 ```yaml
 configuration:
   resources:
     - type: "vertex-configuration"
       name: "Google Vertex AI configuration"
       configuration:
-        url: "{{ secrets.vertex-ai.url }}"
-        # use triple quotes in order to turn off escaping
-        serviceAccountJson: "{{{ secrets.vertex-ai.serviceAccountJson }}}"
+        url: "https://us-central1-aiplatform.googleapis.com"
+        region: "us-central1"
+        project: "my-gcp-project"
+        # Authentication is done either via service account JSON
+        serviceAccountJson: "{\"type\":..........}"
```

Comment:
it is better to use a secrets here, and show how to use the <file:xxx> syntax in the secrets (you can also add a reference to the page in which we talk about <file:xxx>